src/DynamoCoreWpf/Views/PackageManager/Controls/ControlColorsAndBrushes.xaml

```diff
@@ -0,0 +1,152 @@
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:ui="clr-namespace:Dynamo.UI">
+
+    <ResourceDictionary.MergedDictionaries>
+        <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoColorsAndBrushesDictionaryUri}" />
+        <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+    </ResourceDictionary.MergedDictionaries>
+
+    <!-- Colors -->
+    <Color x:Key="MidGrayColor">#999999</Color>
```

Comment:
Some of the values here are also present in the main colors.xaml file, can we just use that to avoid duplication.

Reply:
Removed MidGrayColor from styles